gnd/src/main/java/com/google/android/gnd/ui/home/mapcontainer/MapContainerViewModel.java

```diff
@@ -71,12 +76,16 @@
 @SharedViewModel
 public class MapContainerViewModel extends AbstractViewModel {
 
+  /**
+   * DISTANCE_THRESHOLD check if the first vertex of polygon is within 10 meters of distance from
+   * the current pointing location.
```

Comment:
nit: The comment doesn't provide any benefits to the developer. How about "Minimum distance (in metres) between points to be considered as non-overlapping"?